### PR TITLE
Implementing a new TaskMonitor and Tasks endpoint

### PR DESCRIPTION
**Pull request by GustavoHennig**
Added new TaskMonitor that replaces the existent activity module; (wait_for_task functions...);
Added support to /rest/tasks endpoint. (resource/activity/tasks);
Minor fixes.

### CODE REVIEW

File: hpOneView/oneview_client.py [hunk 1]
```diff
@@ -74,7 +76,7 @@ def __init__(self, config):
         # TODO: Implement: con.set_trusted_ssl_bundle(args.cert)
 
     @classmethod
-    def from_json(cls, file_name):
+    def from_json_file(cls, file_name):
```
Comment (danielreed) [inline hpOneView/oneview_client.py]: This is a better name!

File: hpOneView/resources/task_monitor.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        return int(time.time())
+
+    def wait_for_task(self, task, timeout=60):
+        """
+        Wait for task execution and return associated resource
+        Args:
+            task: task dict
+            timeout: timeout in seconds
+
+        Returns: associated resource when creating or updating; Trues when deleting
```
Comment (danielreed) [inline hpOneView/resources/task_monitor.py]: Is this a typo? Can be fixed in a different pull request.

Comment (GustavoHennig) [inline hpOneView/resources/task_monitor.py]: Yes, it's a typo. :disappointed: It will be fixed in another pull request
